src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/ArmBase.cs

```diff
@@ -43,6 +43,16 @@ public abstract class Arm64
             ///   A64: CLZ Xd, Xn
             /// </summary>
             public static int LeadingZeroCount(ulong value) => LeadingZeroCount(value);
+
+            /// <summary>
+            ///   A64: RBIT Xd, Xn
+            /// </summary>
+            public static long ReverseElementBits(long  value) => ReverseElementBits(value);
```

Comment:
Do we really need this as intrinsic? It is 100% redundant with `BinaryPrimitives.ReverseEndianness`.

Reply:
It's the way one would usually implement a `TrailingZeroCount` on A64, so I guess if `BinaryPrimitives.ReverseEndianness` lowers to `rbit` then it may be redundant, but that means intrinsics users need to know that it does. If there a `BinaryPrimitives.ReverseEndianness` for Vectors as well?

Reply:
We are going to need explicit JIT support for that anyways. Other than the additional API being exposed, is there any harm to exposing this and then having `BinaryPrimitives.ReverseEndianness` either implemented in managed code using this or imported/lowered to the `ArmBase.ReverseElementBits` intrinsic?

Reply:
Also, I don't think this is equivalent. Reverse Endianness does a byte swap. This does a bit reversal.

e.g.
```
ReverseEndianness: `0x0A0D` -> `0x0D0A`
ReverseElementBits: `0x0A0D` -> `0xB050`
```

Reply:
Yes, is different than Reverse Endianness; alas there is no fast way to do it on x86/x64 :(

Reply:
Ah ok. Ignore my comment then.